src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/ECDsaCng.cs

```diff
@@ -151,6 +151,21 @@ private void AcceptImport(CngPkcs8.Pkcs8Response response)
 
         public override bool TryExportPkcs8PrivateKey(Span<byte> destination, out int bytesWritten)
         {
+            const CngExportPolicies Exportable = CngExportPolicies.AllowPlaintextExport | CngExportPolicies.AllowExport;
+            bool encryptedOnlyExport = (Key.ExportPolicy & Exportable) == CngExportPolicies.AllowExport;
```

Comment:
It would be nice if this could be put in just one place.

The best I can come up with is the CngPkcs8 partial that's not in Common (though since we no longer have .Cng split out, either partial might be fine):

```C#
internal static bool AllowsOnlyEncryptedExport(CngKey key)
{
    const CngExportPolicies Exportable = CngExportPolicies.AllowPlaintextExport | CngExportPolicies.AllowExport;
    return (key.ExportPolicy & Exportable) == CngExportPolicies.AllowExport;
}
```

Just because having the same const and bitmask copied over and over makes me nervous.